maa.jwt.verifier.sevsnp/TrustedValues.cs

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using static maa.jwt.verifier.sevsnp.CoseSign1;
+
+namespace maa.jwt.verifier.sevsnp
+{
+    public static class TrustedValues
+    {
+        // -----------------------------------------------------------------------------------
+        // Trusted MAA public root key
+        // -----------------------------------------------------------------------------------
+
+        /// <summary>
+        /// Microsoft Azure Attestation (MAA) root public key used to validate SEV-SNP report signatures.
+        /// Contact the MAA team for updates or rotated keys.
+        /// </summary>
+        public const string MaaRootKey = @"-----BEGIN PUBLIC KEY-----
```

Comment:
What is this?  MAA has a single active signing key, and it's different in each region.

Reply:
removed